tests/conftest.py

```diff
@@ -1714,6 +1714,26 @@ def enum_rand_one_frontend_asic_index(request):
     return request.param
 
 
+@pytest.fixture(scope='module')
+def enum_upstream_dut_hostname(duthosts, tbinfo):
+    for a_dut in duthosts.frontend_nodes:
+        minigraph_facts = a_dut.get_extended_minigraph_facts(tbinfo)
+        minigraph_neighbors = minigraph_facts['minigraph_neighbors']
+        if tbinfo["topo"]["type"] == "t0":
```

Comment:
This can be outside the loop ?

Reply:
Hi @judyjoseph, Thanks for reviewing. Corrected the loop now.